### DISCUSSION TITLE
request api /sb/targets/categories/{categoryRefinementId}/refinements Could not find resource for full path

### DISCUSSION THREAD

**Discussion by LionSniper** [amzn/ads-advanced-tools-docs#249] [Q&A] [answered]
first, i request api "/sb/recommendations/targets/category" to get some category id, and then i use this category id to request api "/sb/targets/categories/{categoryRefinementId}/refinements", like: "https://advertising-api-eu.amazon.com/sb/targets/categories/322086011/refinements"

but it will response to me: {"code":"NOT_FOUND","details":"Could not find resource for full path: http://advertising-api-eu.amazon.com/sb/targets/categories/322086011/refinements"}

please tell me how to resolve it, thanks!

**Comment by SylvieManx** [comment]
This is indeed a bug

LionSniper ***@***.***> 于2024年4月18日周四 11:48写道：

**Comment by LionSniper** [comment]
@elizafarley

**Comment by elizafarley** [comment, marked as answer]
After testing this out, I was able to make the call work by including the `Accept` header set to `application/vnd.sbtargeting.v4+json`. Please try that and see if it works!

```
curl --location 'https://advertising-api.amazon.com/sb/targets/categories/3206324011/refinements' \
--header 'Amazon-Advertising-API-Scope: xxxxxxx' \
--header 'Authorization: xxxxxxx' \
--header 'Amazon-Advertising-API-ClientId: amzn1.application-oa2-client.xxxxxxx' \
--header 'Accept: application/vnd.sbtargeting.v4+json' \
```
